### PR TITLE
Add Keystore and Truststore information to the readme.md

### PR DESCRIPTION
**Pull request by chamindi-a**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,18 @@ Add the configurations to the `custom.yml`. A sample is given below.
 
 Follow the steps mentioned under `docs` directory to customize/create new Ansible scripts and deploy the recommended patterns.
 
+### 3. Including custom Keystore and Truststore
```
Comment (VimukthiPerera) [inline README.md:160]: Shall we add this under customisations?

Comment (VimukthiPerera) [inline README.md:160]: Shall we add this under customisations? WDYT?

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chamindi-a) [reply README.md:160]: @Savidude : let us know your input on this

File: README.md [hunk 2]
```diff
@@ -157,6 +157,18 @@ Add the configurations to the `custom.yml`. A sample is given below.
...
+If custom keystores and truststores are needed to be added, uncomment the below list in the yml file
+# security_file_list:
+#   - { src: '{{ security_file_location }}/wso2am-analytics/client-truststore.jks',
+#       dest: '{{ carbon_home }}/resources/security/client-truststore.jks' }
+#   - { src: '{{ security_file_location }}/wso2am-analytics/wso2carbon.jks',
+#       dest: '{{ carbon_home }}/resources/security/wso2carbon.jks' }
```
Comment (VimukthiPerera) [inline README.md:166]:
```suggestion
```
# security_file_list:
#   - { src: '{{ security_file_location }}/wso2am-analytics/client-truststore.jks',
#       dest: '{{ carbon_home }}/resources/security/client-truststore.jks' }
#   - { src: '{{ security_file_location }}/wso2am-analytics/wso2carbon.jks',
#       dest: '{{ carbon_home }}/resources/security/wso2carbon.jks' }
```
```

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -157,6 +157,13 @@ Add the configurations to the `custom.yml`. A sample is given below.
 
 Follow the steps mentioned under `docs` directory to customize/create new Ansible scripts and deploy the recommended patterns.
 
+### 3. Including custom Keystore and Truststore
+If custom keystores and truststores are needed to be added, uncomment the below list in the yml file
+
```
Comment (VimukthiPerera) [inline README.md:162]:
```suggestion
  # security_file_list:
  #   - { src: '{{ security_file_location }}/wso2am-analytics/client-truststore.jks',
  #       dest: '{{ carbon_home }}/resources/security/client-truststore.jks' }
  #   - { src: '{{ security_file_location }}/wso2am-analytics/wso2carbon.jks',
  #       dest: '{{ carbon_home }}/resources/security/wso2carbon.jks' }
```

Comment (VimukthiPerera) [inline README.md:162]:
```suggestion
# security_file_list:
#   - { src: '{{ security_file_location }}/wso2am-analytics/client-truststore.jks',
#       dest: '{{ carbon_home }}/resources/security/client-truststore.jks' }
#   - { src: '{{ security_file_location }}/wso2am-analytics/wso2carbon.jks',
#       dest: '{{ carbon_home }}/resources/security/wso2carbon.jks' }
```

File: README.md (hunk 1 above)
Comment (chamindi-a) [reply README.md]: Changed the content as needed

**Review by VimukthiPerera — APPROVED** [review]

**Pull request merged**